WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingTagsViewModel.kt

```diff
@@ -0,0 +1,66 @@
+package org.wordpress.android.ui.posts
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import org.wordpress.android.R
+import org.wordpress.android.modules.BG_THREAD
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.viewmodel.Event
+import org.wordpress.android.viewmodel.ScopedViewModel
+import javax.inject.Inject
+import javax.inject.Named
+
+private const val THROTTLE_DELAY = 500L
+
+class PrepublishingTagsViewModel @Inject constructor(
+    private val getPostTagsUseCase: GetPostTagsUseCase,
+    private val updatePostTagsUseCase: UpdatePostTagsUseCase,
+    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher
+) : ScopedViewModel(bgDispatcher) {
+    private var isStarted = false
+    private lateinit var editPostRepository: EditPostRepository
+
+    private val _navigateToHomeScreen = MutableLiveData<Event<Unit>>()
+    val navigateToHomeScreen: LiveData<Event<Unit>> = _navigateToHomeScreen
+
+    private val _dismissBottomSheet = MutableLiveData<Event<Unit>>()
+    val dismissBottomSheet: LiveData<Event<Unit>> = _dismissBottomSheet
+
+    private val _dismissKeyboard = MutableLiveData<Event<Unit>>()
+    val dismissKeyboard: LiveData<Event<Unit>> = _dismissKeyboard
+
+    private val _updateToolbarTitle = MutableLiveData<UiString>()
```

Comment:
NitPick: `update` evokes me that it's a single event (`Event`). I'd consider renaming it to something like `toolbarTitleUiState`, wdyt?

Reply:
Yes, sure. Let me update it to that 🙏